infrastructure/resources/timer_functions.tf

```diff
@@ -239,6 +240,7 @@ resource "azurerm_windows_function_app_slot" "nbs_mya_timer_func_app_preview" {
     "AzureWebJobs.ProposeAvailabilityChangeFunction.Disabled"              = true
     "AzureWebJobs.NotifyBookingAutoCancelled.Disabled"                     = true
     "AzureWebJobs.TriggerAutoCancelledBookings.Disabled"                   = true
+    "AzureWebJobs.SendAutoCancelledBookings.Disabled"                      = var.auto_cancelled_bookings_enabled
```

Comment:
I think you want this inverted - this is currently false on prod but true on dev, so will be enabled on prod and disabled on dev I think